admin/sql/create_tables.sql

```diff
@@ -104,4 +104,26 @@ CREATE TABLE follow_list (
 );
 ALTER TABLE follow_list ADD CONSTRAINT follow_list_name_creator_key UNIQUE (name, creator);
 
+CREATE TABLE recommendation.cf_recording (
```

Comment:
This file should be sorted on table name -- this wasn't quite done when you started working with it, but would you mind putting them in order?